test/Blurb.spec.js

```diff
@@ -20,47 +17,61 @@ describe('A blurb', function() {
         await page.waitFor(SEL_CONTAINER);
         expect(await page.$(SEL_CONTAINER)).to.exist;
     });
-
     it('should have the style class "notice"', async function() {
         await page.waitFor(SEL_NOTICE);
         expect(await page.$$(SEL_NOTICE)).to.have.lengthOf(1);
     });
-
     it('should have the style class "negativeBias"', async function() {
         await page.waitFor(SEL_NEGATIVEBIAS);
-        //expect(await page.$$(SEL_NEGATIVEBIAS)).to.have.lengthOf(1);
+        expect(await page.$$(SEL_NEGATIVEBIAS)).to.have.lengthOf(1);
```

Comment:
Glad you caught this too! We should not have any code commented out in our `master` branch.